persistent/Database/Persist/Quasi/Internal.hs

```diff
@@ -98,8 +99,16 @@ parseFieldType t0 =
             PSDone -> PSSuccess (front []) t
             -- _ ->
 
+newtype ToFKName = ToFKName
```

Comment:
As an exposed type, let's put a doc comment on it:

```suggestion
-- | A function for converting a table and column name into a constraint name.
-- 
-- @since 2.13.0.0
newtype ToFKName = ToFKName
```